Cargo.toml

```diff
@@ -24,7 +23,7 @@ name = "example"
 path = "examples/main.rs"
 
 [dependencies]
-base64  = "0.10"
-xml-rs  = "0.8"
-libflate = "0.1.18"
-zstd = { version = "0.9", optional = true }
+base64  = "0.13.0"
+xml-rs  = "0.8.4"
```

Comment:
Fixing the minor version is not a good idea. These won't be breaking and we'll get them for free if we just do `0.8`.

Reply:
The only difference between putting `0.8` or `0.8.4` is in the minimum version, in both cases the upper bound is `<0.9.0`. These version bumps are just done using `cargo upgrade` (after installing `cargo-edit`).

Reply:
I see.